packages/app/background-jobs-common/src/background-job-processor/managers/QueueManager.ts

```diff
@@ -85,10 +85,12 @@ export class QueueManager<
     return this._queues[queueId]
   }
 
-  public async start(queueIdsToStart?: string[]): Promise<void> {
+  public async start(enabled: string[] | boolean = true): Promise<void> {
```

Comment:
do we need to update anything in readme?

Reply:
Current readme does not dive deep and has no details on different `start()` options. But all code examples are up to date, using no arg which defaults to `enabled = true` 👍